hooks/pre-command

```diff
@@ -80,8 +86,14 @@ elif [[ "${MULTIPLE_DOWNLOADS}" == "true" ]]; then
           if ! [[ -d $(dirname "${!dest_env_var}") ]]; then
             mkdir -p "$(dirname "${!dest_env_var}")"
           fi
-          echo "~~~ Moving [${source}] to [${!dest_env_var}]..."
-          mv "${source}" "${!dest_env_var}"
+          # Handler for extract artifacts
+          if [[ "${BUILDKITE_PLUGIN_ARTIFACTS_DOWNLOAD_EXTRACT:-false}" =~ ^(true|on|1)$ ]] ; then
```

Comment:
for multiple downloads - we need to be looking at the indexed parameter; similar to the way that `source_env_var` is being handled.

Reply:
Yeah, I fixed it.